gapic/src/main/com/google/gapid/perfetto/views/TraceConfigDialog.java

```diff
@@ -226,10 +226,47 @@ public static String getConfigSummary(Settings settings, Device.PerfettoCapabili
 
     boolean largeBuffer = false;
     if (settings.perfettoVulkanCPUTiming) {
+      String s = "";
```

Comment:
If you want to build up a string, use a `StringBuilder`. However, the code will be more readable if you build up a `List<String>` and then use `Joiner.on(":").join(...)`.